.github/workflows/build.yml

```diff
@@ -15,6 +15,7 @@ on:
       dependency-identifiers:
         type: string
       msbuild-dependency-target:
+        default: CKANInstall
```

Comment:
I think I'd prefer to leave this empty, and make the ckan-setup and install-dependencies steps conditional on them.  Then the users of this workflow will opt-in by adding a single line in their workflow that calls this one.